app/src/main/res/values/strings.xml

```diff
@@ -55,7 +55,7 @@
   <string name="patiententry_full_name">Patient\'s name</string>
   <string name="patiententry_phone_number">Phone number</string>
   <string name="patiententry_bangladesh_national_id">National ID number</string>
-  <string name="patiententry_ethiopia_national_id">National ID</string>
+  <string name="patiententry_ethiopia_medical_record_number">Medical Record Number</string>
```

Comment:
If we are changing the key here, we should be deleting the strings in the resource files for other languages since Transifex will not pick it up as a new string and mark them as translated, which will be incorrect since the source string has changed.